coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -1101,13 +1101,11 @@ func (s *server) notifyWorkspaceBuildFailed(ctx context.Context, workspace datab
 		return // failed workspace build initiated by a user should not notify
 	}
 	reason = string(build.Reason)
-	initiator := "autobuild"
 
 	if _, err := s.NotificationsEnqueuer.Enqueue(ctx, workspace.OwnerID, notifications.TemplateWorkspaceAutobuildFailed,
 		map[string]string{
-			"name":      workspace.Name,
-			"initiator": initiator,
```

Comment:
I found out that `TemplateWorkspaceAutobuildFailed` was not using the initiator label in the template body so I just removed it.